### PR TITLE
Opengl compatibility for :class:`~.VDict`, :class:`~.Axes.get_line_graph` and :class:`~.FocusOn`.

### PR DESCRIPTION
**Pull request by hydrobeam**
## Motivation

More OpenGL action enabled

## Explanation for Changes

Where there was a direct call to `Vmobject`, add a check to create an `OpenGLVmobject` instead, and add `metaclass=OpenGLCompatiblity` to the definition of the class.

## Documentation Reference
 
N/A

## Testing Status

code for testing VDict / get_line_graph
```py
class show(Scene):
    def construct(self):
        ax = Axes()
        gr = ax.get_line_graph(np.arange(0, 4, 0.2), np.arange(2, 4, 0.5), add_vertex_dots=False)
        self.add(ax, gr)
```
(alternatively, sample code for VDict in the docs can be used to test: [link](https://docs.manim.community/en/stable/reference/manim.mobject.types.vectorized_mobject.VDict.html?highlight=vdict#shapeswithvdict) )

code for testing FocusOn

```py
class show2(Scene):
    def construct(self):
        self.play(FocusOn(ORIGIN))
```

## Further Comments

I also wanted to make `ImageMobject` work in this PR, but it was getting some weird results when porting+adjusting manimgl code.

These seem like the last of the simple compatibility fixes. 

## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [x] I have written a descriptive PR title (see top of PR template for examples)
- [x] I have written a changelog entry for the PR or deem it unnecessary
- [x] My new functions/classes either have a docstring or are private
- [x] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [x] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

File: manim/animation/indication.py [hunk 1]
```diff
@@ -104,7 +104,13 @@ def __init__(
...
         # Initialize with blank mobject, while create_target
         # and create_starting_mobject handle the meat
-        super().__init__(VMobject(), run_time=run_time, remover=remover, **kwargs)
+        if config.renderer == "opengl":
+            from manim.opengl import OpenGLVMobject
+
+            vmob_class = OpenGLVMobject
+        else:
+            vmob_class = VMobject
+        super().__init__(vmob_class(), run_time=run_time, remover=remover, **kwargs)
```
Comment (TRoboto) [inline manim/animation/indication.py:113]: I think you can just use `VGroup` instead.

File: manim/mobject/coordinate_systems.py [hunk 2]
```diff
@@ -1243,6 +1243,16 @@ def construct(self):
...
+
+        if config.renderer == "opengl":
+            from manim.opengl import OpenGLVMobject
+
+            graph_class = OpenGLVMobject
+        else:
+            graph_class = VMobject
+
+        graph = graph_class(color=line_color, **kwargs)
+
```
Comment (TRoboto) [inline manim/mobject/coordinate_systems.py:1255]: Same here.

**Review by TRoboto — CHANGES_REQUESTED** [review]

File: manim/animation/indication.py (hunk 1 above)
Comment (hydrobeam) [reply manim/animation/indication.py]: Good idea 👍

**Review by jsonvillanueva — APPROVED** [review]
LGTM

**Pull request merged**